File: core/src/com/oni/donjon/Resources.java
package com.oni.donjon;

import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.utils.I18NBundle;

/**
 * @author Daniel Chesters (on 27/05/14).
 */
public class Resources {
    public static final I18NBundle bundle = I18NBundle.createBundle(Gdx.files.internal("i18n/messages"));

    private Resources() {
    }
}

File: core/src/main/kotlin/com/oni/donjon/Resources.kt
package com.oni.donjon

import com.badlogic.gdx.Gdx
import com.badlogic.gdx.utils.I18NBundle
import com.badlogic.gdx.utils.I18NBundle.createBundle

/**
 * @author Daniel Chesters (on 30/05/2017).
 */
object Resources {
    val BUNDLE: I18NBundle = createBundle(Gdx.files.internal("i18n/messages"))
}
